back/src/main/resources/application.properties

```diff
@@ -22,5 +22,5 @@ jwt-auth-server.keyAlias=jwt
 jwt-auth-server.myFront=http://localhost:3000
 jwt-auth-server.clientId=my-client-app
 
-jwt-auth-server.accessTokenValiditySeconds=1800
+jwt-auth-server.accessTokenValiditySeconds=180000
 jwt-auth-server.refreshTokenValiditySeconds=86400
```

Comment:
you can extract clientId  + token duration in env variables. This permits to have a specific configuration in eclipse / intelliJ/ whatever IDE you use, and pass another values in Jenkins / XLDeploy / Script / Whatever deployment solution you use when deploying in production.